src/BuiltInTools/dotnet-watch/LaunchSettingsProfile.cs

```diff
@@ -18,13 +18,15 @@ public class LaunchSettingsProfile
 
         public string? CommandName { get; set; }
 
+        // TODO: Add support for commandLineArgs defined in the launch profile
```

Comment:
You can do this today?

Reply:
Actually, after testing it locally it seems only Visual Studio honors it and not `dotnet run`.